library/pkcs5.c

```diff
@@ -230,9 +230,10 @@ int mbedtls_pkcs5_pbkdf2_hmac( mbedtls_md_context_t *ctx, const unsigned char *p
 
     memset( counter, 0, 4 );
     counter[3] = 1;
-
+#if defined(MBEDTLS_HAVE_INT64)
```

Comment:
The macro `MBEDTLS_HAVE_INT64` is defined in `mbedtls/bignum.h`, but this header file is not included in this source file.